File: typescipt/src/compiler/JavaScript/builtInFunctions/daysFunctionCall.ts
import {EndStartDateFunctionCall} from "./endStartDateFunctionCall";
import {DaysFunction} from "../../../language/expressions/functions/daysFunction";

export class DaysFunctionCall extends EndStartDateFunctionCall {
  protected override className = "BuiltInDateFunctions";
  protected override methodName = "days";

  constructor(functionNode: DaysFunction) {
    super(functionNode);
  }
}

File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/BuiltInFunctions/DaysFunctionCall.cs
using Lexy.Compiler.Language.Expressions.Functions;
using Lexy.RunTime;

namespace Lexy.Compiler.Compiler.CSharp.BuiltInFunctions;

internal class DaysFunctionCall : EndStartDateFunctionCall<DaysFunction>
{
    protected override string ClassName => nameof(BuiltInDateFunctions);
    protected override string MethodName => nameof(BuiltInDateFunctions.Days);
}